vnext/docs/ConsumingRNW.md

```diff
@@ -38,10 +38,12 @@
 
 ## Running a React Native Windows App
 
-*Note: Make sure Chrome is launched and running before running a React Native Windows app.*
+*Note: Make sure a browser is launched and running before running a React Native Windows app.*
 
 - Without Visual Studio
 
+:exclamation: **We are seeing issues with msbuild with the below command. This is being investigated in [Issue 3263](https://github.com/microsoft/react-native-windows/issues/3263). Until this is fixed, please use the With Visual Studio option below.**
```

Comment:
Not all customer hit this.
It's better to add a troubleshooting page, and both ConsumingRNW.md and BuildingRNW.md refer to it.
In the troubleshooting page. We had things like this:
1. Title 'error C2373: '__builtin_clz': redefinition; different type modifiers' whn run `react-native run-windows`
2. Description
  This happens on VS16.3.1 and msbuild version abc when building Folly
  msbuild fail with error xyz but vs build success...
3. Workaround
- use other version of VS
- deploy the app from VS. Procedure abc